### ISSUE TITLE
bluebirdCoroutines – _interopRequireWildcard used before it's defined

### ISSUE THREAD

**Issue by nthtran** [babel/babel#1289]
When using `bluebirdCoroutines` `babel` generates the following code:

```js
'use strict';

var _bluebird2 = require('bluebird');

var _bluebird3 = _interopRequireWildcard(_bluebird2);

var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };

var _interopRequireWildcard = function (obj) { return obj && obj.__esModule ? obj : { 'default': obj }; };

...
```

Which results in an `undefined is not a function` error.

**Comment by nthtran** [comment]
Works fine when `runtime` is used:

```js
'use strict';

var _bluebird2 = require('bluebird');

var _interopRequireWildcard = require('babel-runtime/helpers/interop-require-wildcard')['default'];

var _bluebird3 = _interopRequireWildcard(_bluebird2);
```

**Comment by calebboyd** [comment]
+1

**Comment by bucaran** [comment]
+2

**Comment by sebmck** [comment]
Fixed as of 5.1.11, thanks!

Also just a nit but +1s are incredibly unproductive, extremely annoying and don't get bugs fixed faster.
